Rubberduck.Parsing/VBA/DeclarationCaching/DeclarationFinder.cs

```diff
@@ -1034,12 +1034,8 @@ public IEnumerable<Declaration> FindNewDeclarationNameConflicts(string newName,
 
             if (IsEnumOrUDTMemberDeclaration(renameTarget)) 
             {
-                var memberMatches = identifierMatches.Where(idm =>
+                return identifierMatches.Where(idm =>
```

Comment:
I'm unclear how this fixes the issue 4349. This seems to be just refactoring the code without any changes in the code's behavior?